packages/cluster-ui/src/statementsPage/statementsPage.tsx

```diff
@@ -76,22 +79,36 @@ export interface StatementsPageState {
   sortSetting: SortSetting;
   search?: string;
   pagination: ISortedTablePagination;
-  fullScan: boolean;
+  filters?: Filters;
+  activeFilters?: number;
 }
 
 export type StatementsPageProps = StatementsPageDispatchProps &
   StatementsPageStateProps &
   StatementsPageOuterProps &
-  RouteComponentProps<any>;
+  RouteComponentProps<unknown>;
 
 export class StatementsPage extends React.Component<
   StatementsPageProps,
   StatementsPageState
 > {
   activateDiagnosticsRef: React.RefObject<ActivateDiagnosticsModalRef>;
+  searchParams = getSearchParams(this.props.history.location.search);
 
   constructor(props: StatementsPageProps) {
     super(props);
+    const filters: Filters = {
+      app: this.searchParams("app", defaultFilters.app).toString(),
```

Comment:
because the searchParams can return string or boolean (we have both types of filters), if I don't enforce that for the app is indeed a string it complains the type don't match